pkgs/data/fonts/iosevka/comfy.nix

```diff
@@ -23,25 +23,40 @@ let
     sha256 = "1h72my1s9pvxww6yijrvhy7hj9dspnshya60i60p1wlzr6d18v3p";
   };
   privateBuildPlan = src.outPath + "/private-build-plans.toml";
-  overrideAttrs = (attrs: {
-    inherit version;
-
-    meta = with lib; {
-      inherit (src.meta) homepage;
-      description = ''
-        Customised build of the Iosevka typeface, with a consistent
-        rounded style and overrides for almost all individual glyphs
-        in both roman (upright) and italic (slanted) variants.
-      '';
-      license = licenses.ofl;
-      platforms = attrs.meta.platforms;
-      maintainers = [ maintainers.DamienCassou ];
-    };
-  });
   makeIosevkaFont = set:
-    (callPackage ./. { inherit set privateBuildPlan; }).overrideAttrs
-    overrideAttrs;
-in builtins.listToAttrs (builtins.map (set: {
-  name = set;
-  value = makeIosevkaFont set;
-}) sets)
+    let superBuildNpmPackage = buildNpmPackage; in
+    (iosevka.override rec {
+      inherit set privateBuildPlan;
+      buildNpmPackage = args: superBuildNpmPackage
+        (args // {
+          inherit version;
+
+          src = fetchFromGitHub {
+            owner = "be5invis";
```

Comment:
is it possible to override just the `rev` and `hash` fields?

Reply:
I am not sure how this would be done. We're not merging recursively, and even if we were `src` isn't a set so I don't think we could just specify `rev` and `hash` and pick up the rest from the base package. Did you have an idea on how to handle this?

Reply:
Absolutely not. I just thought you might :-). Let's keep it like this then.